src/main/java/com/google/javascript/clutz/DeclarationGenerator.java

```diff
@@ -319,7 +335,7 @@ public int compare(TypedVar o1, TypedVar o2) {
     }
     // extra walk required for inner classes and inner enums. They are allowed in closure,
     // but not in TS, so we have to generate a namespace-class pair in TS.
-    if (isDefault && hasNestedTypes(symbol.getType())) {
+    if (isDefault && hasNestedTypes(symbol.getType()) && !isExtern) {
```

Comment:
Because, it is easier to let the other loop take care of them, since it is going through all the symbols already.

added comment.